collector/src/bin/collector.rs

```diff
@@ -313,6 +313,10 @@ struct LocalOptions {
     #[arg(long)]
     cargo: Option<PathBuf>,
 
+    /// Arguments passed to `cargo --config <value>`.
+    #[arg(long)]
+    cargo_config: Option<Vec<String>>,
```

Comment:
Let's simplify this.
```suggestion
    cargo_config: Vec<String>,
```